test/system/010-list.bats

```diff
@@ -23,22 +23,20 @@ load helpers
 }
 
 @test "ramalama list - json" {
-    #FIXME jq version on mac does not like regex handling
-    skip_if_darwin
     # 'created': ramalama includes fractional seconds, ramalama-remote does not
     tests="
-name              | [a-z0-9A-Z:/]\\\+
-modified          | [0-9]\\\+
-size              | [0-9]\\\+
+name              | test(\"^[a-z0-9A-Z:/]\\+\")
+modified          | tostring | test(\"^[a-z0-9A-Z:/]\\+\")
+size              | tostring | test(\"^[0-9]\\+\$\")
```

Comment:
it checks also now with a $ at the end to ensure the size is only a digit and not '123 MB' or '4 GB'
else the test was passing